packages/connect-examples/new-example/app/components/ui/Input.tsx

```
@@ -0,0 +1,22 @@
+import * as React from "react"
+
+import { cn } from "~/lib/utils"
+
+const Input = React.forwardRef<HTMLInputElement, React.ComponentProps<"input">>(
+  ({ className, type, ...props }, ref) => {
```

ESLint

[error] 'className' is missing in props validation

(react/prop-types)